libs/application-generic/src/usecases/create-workflow/create-workflow.usecase.ts

```diff
@@ -135,6 +119,32 @@ export class CreateWorkflow {
     return storedWorkflow;
   }
 
+  private generateTriggerIdentifier(command: CreateWorkflowCommand) {
+    if (command.triggerIdentifier) {
+      return command.triggerIdentifier;
+    }
```

Comment:
If provided, we will skip the internal generation. This is somewhat of a risk area, which is why I added a comment on the command parameter.